src/status_im/contexts/preview/quo/list_items/community_list.cljs

```diff
@@ -95,7 +95,7 @@
                              :unread-count        5})]
     (fn []
       [preview/preview-container {:state state :descriptor (descriptors @state)}
-       [quo/community-list-item
+       [quo/community-list
```

Comment:
a comment probably a little unrelated to this pr, as for this `community-list` component, I would have preferred if it was in the community ns `status-im.contexts.preview.quo.community` instead of the list-items ns `status-im.contexts.preview.quo.list-items`. Its like its the only community component outside this community component ns.

the `community-membership-list-view` is also a list-item community component and we have this component is the community ns wdyt?

Reply:
Totally agree. Did that in the last commit.